src/main/modules/app-insights/index.ts

```diff
@@ -6,13 +6,37 @@ declare class AppInsightsConfiguration {
   roleName: string
 }
 
+function hideUuidInUrlIfNotStaticFile (url: string): string {
+  let fileRegexp = new RegExp('(\\..{2,5}$)')
```

Comment:
what's this regexp for?

Reply:
Checking if URL is static file, ending with dot and 2 to 5 letters extension. So it prevents us not to rename files.

Reply:
O